addons/mail/static/src/utils.js

```diff
@@ -1,6 +1,6 @@
 /** @odoo-module */
 
-import { onMounted, onPatched, onWillPatch, onWillUnmount, useRef } from "@odoo/owl";
+import { onMounted, onPatched, onWillPatch, onWillUnmount, reactive, useEffect, useExternalListener, useRef, useState } from "@odoo/owl";
```

Comment:
Missing Prettier (should have turned into multi-line).
Make sure to properly configure prettier on your machine.

Reply:
Yes, but I thought the multiline was not wanted? Because it changed this into multiline but I didn't see this in other files？